harmonica/forward/prism.py

```diff
@@ -212,6 +234,9 @@ def jit_prism_gravity(coordinates, prisms, density, kernel, out):
     """
     # Iterate over computation points and prisms
     for l in prange(coordinates[0].size):
+        # Update progress bar if called
+        if progress_proxy is not None:
```

Comment:
Thinking about some little optimizations.

I imagine that checking if `progress_proxy` is `None` on each iteration is slower than checking if a boolean is True or False. So, I'm thinking we could do something like:

```python
# Check if we need to update the progressbar on each iteration
update_progressbar = (progress_proxy is not None)
# Iterate over computation points and prisms
    for l in prange(coordinates[0].size):
        for m in range(prisms.shape[0]):
            ...
        # Update progress bar if called
        if update_progressbar:
            progress_proxy.update(1)
```

I'm not sure if it would run faster, but might worth experimenting maybe.
cc @leouieda

Reply:
That seems like a better option. I've just done a quick test and it appears to be faster, ~7%, but that was just a single test.

Reply:
Awesome! I was expecting just a minor improvement, so this is great.